process/process_darwin_arm64.go

```diff
@@ -120,22 +120,25 @@ type Sigacts struct{}
 
 type ExternProc struct {
 	P_un        [16]byte
-	P_vmspace   *Vmspace
-	P_sigacts   *Sigacts
+	P_vmspace   uint64
```

Comment:
Is the "non-v3" version still maintained, or should this one be kept as-is? (I see https://github.com/shirou/gopsutil/commit/20c3ddbfe528ec30c19d786875383d7753702a9b also modified both, so probably ok?)

Reply:
v2 is still maintained, but may be deprecated soon (before end of 2021 if I remember @shirou's previous statement)